### DISCUSSION TITLE
Does lego support alias mode?

### DISCUSSION THREAD

**Discussion by Mmx233** [go-acme/lego#1614] [Q&A]
CNAME all domains' `_acme-challenge` to an domain and add TXT record to the domain to verify. I cannot find such option in lib. And I also tryed to make a DNS provider struct which return a same domain but this cannot work.

**Comment by ldez** [comment]
Hello,

you can use: https://go-acme.github.io/lego/dns/#experimental-features

**Reply by Mmx233** [reply]
But there are no options to config the domain used to add dns recode. Should I make a modified provider to do this again?

**Reply by Mmx233** [reply]
Or it will follow the CNAME and find out which domain to use automaticly?
